flex/flex.h

```diff
@@ -0,0 +1,40 @@
+// Compiler for PHP (aka KPHP)
+// Copyright (c) 2021 LLC «V Kontakte»
+// Distributed under the GPL v3 License, see LICENSE.notice.txt
+
+#pragma once
+
+#include "common/wrappers/string_view.h"
+
+struct lang;
+
+struct node {
```

Comment:
What do you think of renaming `node` to `flex_node` and `lang` to `flex_lang`? It's a bit weird that such general names exist in a global scope

Reply:
I agree with you, but the problem is It will break backward compatibility. In that case we will need to update flex, kphp and vkext simultaneously somehow
Current version preserves backward compatibility and we can simply update flex at first, and then kphp and vkext